### ISSUE TITLE
Contrast ratio for text and icons do not meet WCAG standards.

### ISSUE THREAD

**Issue by RyanScottLewis** [amberframework/amberframework.org#31]
Very low contrast ratios can be limiting to users with visual impairment due to their [contrast sensitivity](https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Overview_of_Low_Vision#Contrast_and_Light_Sensitivity).

Summary about contrast ratios from [Web Content Accessibility Guidelines (WCAG) 2.0](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast):

> Level AA: The visual presentation of text and images of text has a contrast ratio of at least 4.5:1
> Level AAA: The visual presentation of text and images of text has a contrast ratio of at least 7:1:1

Items which meet neither AA nor AAA WCAG 2.0 specifications:
* Top navigation bar logo/hovered links are [2.21:1](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=F4994B), although the logo has no contrast requirements according to WCAG.
* Top navigation bar unhovered links are only [1.60:1](https://webaim.org/resources/contrastchecker/?fcolor=FAD5B5&bcolor=F4994B) due to the opacity being turned down for the hover effect.
* The first copy you can read on the homepage directly below hero image ("Bare metal performance") is [1.24:1](https://webaim.org/resources/contrastchecker/?fcolor=e4e7ea&bcolor=FFFFFF).
* The small icons next to the grid items lower on the page ("Generators", "No Configuration", etc) and the "hamburger" icon in the sidebar of `/guides/` is also 1.24:1, although these seem to qualify as incidental or "pure decoration" items according to WCAG.
* The copyright in the footer is [3.61:1](https://webaim.org/resources/contrastchecker/?fcolor=788394&bcolor=F8F8F8).
* Most links are [2.56:1](https://webaim.org/resources/contrastchecker/?fcolor=0FACF3&bcolor=FFFFFF).
* Contribution copy ("Contributing to Amber can be...") is [1.86:1](https://webaim.org/resources/contrastchecker/?fcolor=B5B9BF&bcolor=F8F8F8).
* Every single button with the exception of black text on white background. Each color combo has differing contrast ratios, but all failing AA.

The rest of the copy from what I saw (in `/guides`, for example) has a contrast ratio of [7.69:1](https://webaim.org/resources/contrastchecker/?fcolor=535353&bcolor=FFFFFF), which passes WCAG 2.0 AAA specifications.

Other contrast issues can be seen via [WAVE on webaim.org](http://wave.webaim.org/report#/https://amberframework.org/) (be sure to select "Contrast" in the sidebar's tabs.)

**Comment by RyanScottLewis** [comment]
This was accidentally automatically closed due to the phrase "fix #31" being included within the description of #34.

**Comment by faustinoaq** [comment]
Yeah, I think this issue isn't solved, We should reopen it

**Comment by faustinoaq** [comment]
@RyanScottLewis I fixed almost all styles errors [0] but Contrast is a bit difficult to fix without changing Amber look and feel. I tried multiple colors:

![0 0 0 0-3000- 3](https://user-images.githubusercontent.com/3067335/31494042-45e4072e-af17-11e7-909f-aacd8f4beb01.png)

![0 0 0 0-3000- 2](https://user-images.githubusercontent.com/3067335/31494044-46b7ec4c-af17-11e7-8050-31cbe8b9ee82.png)

![0 0 0 0-3000- 1](https://user-images.githubusercontent.com/3067335/31494068-5ecbc98e-af17-11e7-8dcd-b9ff733af460.png)

I'm using this colors:

- navbar: `#cb8040`
- big blue button: `#0facf3`
- small text: `#535353`
- anchor links: `#1a79a3`
- .text-light: `#5c697e`

These are the main constrast issues I found in [0]

[0]] - http://wave.webaim.org/report#/https://amberframework.org/

@eliasjpr @drujensen @elorest WDYT ?

**Comment by drujensen** [comment]
@faustinoaq not my strength.  I still prefer a box of 8 crayons because the 64 is too complicated.  :-)

**Comment by faustinoaq** [comment]
@RyanScottLewis I did some little contrast enhancements on #37

**Comment by RyanScottLewis** [comment]
@faustinoaq I made some changes to a branch on my fork which allows the white on orange theme to pass AA spec. Menu items are sized to WCAG large text size with full opacity and all references to the older orange color were replaced with the `$main-orange` variable. Note this commit doesn't update the copy on the [Media page](https://amberframework.org/media).

https://github.com/amberframework/amberframework.org/compare/master...RyanScottLewis:main-orange-contrast

**Comment by faustinoaq** [comment]
Hi @RyanScottLewis Thanks so much!

I did a PR https://github.com/amberframework/amberframework.org/pull/41 and a preview on https://amberframework-pr-41.herokuapp.com/ 

Please, I need you run `npm install; npm run release` to generte new `main.bundle.css`
